test/integration/diag_nodes_test.go

```diff
@@ -16,7 +16,8 @@ import (
 	testserver "github.com/openshift/origin/test/util/server"
 )
 
-func TestDiagNodeConditions(t *testing.T) {
+// TODO FIX THIS
+func DISABLETestDiagNodeConditions(t *testing.T) {
```

Comment:
@liggitt @smarterclayton would you trade one test (for now) for 8 minutes in the queues?